requirements.txt

```diff
@@ -17,3 +17,4 @@ django-file-resubmit>=0.5.2,<1.0.0
 django-extensions==2.2.8
 easy-thumbnails==2.7
 requests==2.23.0
+sentry-sdk==0.14.3
```

Comment:
Do we want Sentry in the dev environment as well?

Reply:
it was added to `base.py` so it was required.. I think we can take it out from dev